fdbclient/DatabaseConfiguration.cpp

```diff
@@ -158,6 +160,20 @@ void parse(std::vector<RegionInfo>* regions, ValueRef const& v) {
 	}
 }
 
+// Expect a string value like '["a","b","c"]'.
+void parse(std::vector<StringRef>* splits, ValueRef const& v) {
```

Comment:
I suppose changing the name `parse` to something like `parseStringArray` might be better, since the output `std::vector<StringRef>*` is generic, not specialized to a given type.

Reply:
This `parse` function is overloaded for handling many different types. So I'll keep this name to be consistent.